lib/fathead/httplib2/parse.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python2.7
+# -*- coding: utf-8 -*-
+
+import re
+from bs4 import BeautifulSoup
+
+doc_file = "download/libhttplib2.html"
+external_link = "https://httplib2.readthedocs.io/en/latest/libhttplib2.html"
+
+
+# Makes the abstract formatted from a definition line, returns a tuple
+# with the abstract and the source url
+def parse_dl(dl):
+    # extract the link before the signature to crop the pilcrow (¶)
+    link = dl.dt.find("a", {'class': 'headerlink'}).extract()
+    link = '{}{}'.format(external_link, link['href'])
+
+    flags = re.MULTILINE | re.UNICODE
+
+    signature = dl.dt.text
+    signature = re.sub(r'\s', ' ', signature, flags=flags).strip()
+    signature = '<pre><code>{}</code></pre>'.format(signature)
+
+    description = dl.dd.p.text
+    description = re.sub(r'\s', ' ', description, flags=flags).strip()
+    description = '<p>{}</p>'.format(description)
+
+    abstract = '<div class="prog__container">{}{}</div>'.format(signature,
```

Comment:
@Marie-Donnie please use `section` instead of `div` -- we ran into some issues with `div` and had to change it. I've updated the docs accordingly 👍